policyengine_us/variables/gov/states/ga/tax/income/subtractions/ga_retirement_exclusion.py

```diff
@@ -8,8 +8,39 @@ class ga_retirement_exclusion(Variable):
     unit = USD
     definition_period = YEAR
     reference = (
-        "https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download#page=15"
-        "https://dor.georgia.gov/document/document/2022-it-511-individual-income-tax-booklet/download#page=15"
+        "https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download"  # Page 15
+        "https://dor.georgia.gov/document/document/2022-it-511-individual-income-tax-booklet/download"  # Page 15
         "https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=fb5db531-a80f-4790-bddb-eefc8327ef60&config=00JAA1MDBlYzczZi1lYjFlLTQxMTgtYWE3OS02YTgyOGM2NWJlMDYKAFBvZENhdGFsb2feed0oM9qoQOMCSJFX5qkd&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A65D2-CDH3-CGX8-044N-00008-00&pdcontentcomponentid=234186&pdteaserkey=sr1&pditab=allpods&ecomp=8s65kkk&earg=sr1&prid=66f02b0a-c5ae-4162-9535-127751546807"
     )
     defined_for = StateCode.GA
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.ga.tax.income.agi.exclusions.retirement
+        age_younger = (person("age", period) >= p.age.younger) & (
```

Comment:
create a separate `ga_retirement_exclusion_eligible` variable, person-level defined similar to the military one using the younger age limit and head_or_spouse

then define `older = person("age", period) >= p.age.older`

for the future, please create a variable like `age = person("age", period)` if using it multiple times